### ISSUE TITLE
Error in UniqueExcludeCurrentValidator

### ISSUE THREAD

**Issue by hhjse** [evoWeb/sf_register#119]
I get an Error, when I use the UniqueExcludeCurrentValidator

PHP Warning: sprintf(): Too few arguments in ... LocalizationUtility

When I add the arguments it works

At Line 87
```
        if ($this->userRepository->countByField($this->propertyName, $value)) {
            $this->addError(
                $this->translateErrorMessage(
                    'error_notunique_local',
                    'SfRegister',
                    [$this->translateErrorMessage($this->propertyName, 'SfRegister')]
                ),
                1301599608
            );
        } elseif ($this->options['global'] && $this->userRepository->countByFieldGlobal($this->propertyName, $value)) {
            $this->addError(
                $this->translateErrorMessage(
                    'error_notunique_global',
                    'SfRegister',
                    [$this->translateErrorMessage($this->propertyName, 'SfRegister')]
                ),
                1301599619
            );
        }
```

**Comment by garbast** [comment]
Feel free to hand in a pull request.

**Comment by garbast** [comment]
Resolved in release 10.0.2
